packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -639,6 +639,8 @@ The following example expressions are supported for events with the `filter` que
 | `actor.id eq ":id"`                          | Events that are published with a specific actor ID                                      |
 
 > **Note:** SCIM filter expressions can't use the `published` attribute since it may conflict with the logic of the `since`, `after`, and `until` query params.
+> Furthermore, using the `co` (contains) operator with the `debugContext.debugData.url` and `debugContext.debugData.requestUri` attributes is not supported and will
+> result in a 400 API response.
```

Comment:
Maybe we don't need to specify the error response if the expression is not supported?

Reply:
That's actually the reason why we do show it.  If you look at the examples in this section, they follow the same pattern. @dandennhardt-okta since this was your suggestion, care to comment?

Reply:
sounds good, i'll add the note back in